cluster-autoscaler/cloudprovider/azure/azure_scale_set_test.go

```diff
@@ -37,9 +37,10 @@ func newTestScaleSet(manager *AzureManager, name string) *ScaleSet {
 		azureRef: azureRef{
 			Name: name,
 		},
-		manager: manager,
-		minSize: 1,
-		maxSize: 5,
+		manager:           manager,
+		minSize:           1,
+		maxSize:           5,
+		enableForceDelete: manager.config.EnableForceDelete,
```

Comment:
can we have unit tests for both true/false situation?

Reply:
Done